lib/app.js

```diff
@@ -72,9 +72,14 @@ class Container extends Component {
 
   render () {
     const { Component, props, url } = this.props
+    const component = (typeof Component.renderPage === 'function') ? (
+      Component.renderPage({ Component, props, url })
```

Comment:
Why isn't this just:

```js
Component.renderPage(this.props)
```

In other word, why do most of `this.props` get packed into `props`? It seems like to me that `renderPage` should get the same props as `Component` gets + the `Component` itself, which is just the original `this.props`.

Reply:
200% agree! 🔥🔥🔥